### ISSUE TITLE
Definition of "Phrase" in Glossary would be helpful

### ISSUE THREAD

**Issue by kerstinp** [w3c/wcag21#287]
Because of several discussions in the context of SC 3.1.2 Language of Parts WCAG 2.0 it would be helpful to have a clear definition of "phrase" in the glossary of WCAG 2.1.

**Comment by awkawk** [comment]
Kerstin, this seems to be a very common word and the standard definition should suffice:

"a word or group of words forming a syntactic constituent with a single grammatical function"

What is the issue you are thinking about?

**Comment by mapluke** [comment]
I agree. Phrase is only referred to in its standard dictionary sense. Where a common word is intended to mean something very technical and specific (like the meaning of "text" in WCAG 2.0) it is necessary to include a glossary entry. In this case, the word "phrase" is being used with its standard dictionary meaning (which should be understood by anyone who has been taught basic grammar).

**Comment by kerstinp** [comment]
Hi Andrew,

sorry for late commenting. 

Wikipedia for example says: "(...) In linguistic analysis, a phrase is a group of words (or possibly a single word) that functions as a constituent in the syntax of a sentence, a single unit within a grammatical hierarchy. A phrase typically appears within a clause, but it is possible also for a phrase to be a clause or to contain a clause within it."

"In common usage, a phrase is usually a group of words with some special idiomatic meaning or other significance, such as "all rights reserved", "economical with the truth", "kick the bucket", and the like. "

But which usage is meant in WCAG? The common usage or the usage in linguistic analysis? Therefore I feel that an entry in the glossary would be helpful and would stop some discussions in the one way or the other. Probably this issue is more relevant for example in for example german texts than in english texts. In german texts we have depending on the topic of course more words from other languages - especially english ones.

**Comment by awkawk** [comment]
@kerstinp WCAG 2.0 is definitely referring to the "group of words" definition. A group of words with a special idiomatic meaning would also be a phrase, but the group of words doesn't need to have special meaning, it could just be a group of words. e.g. "Tom wants to introduce himself in French, so he says "(whatever hi, I'm Tom is in French)".

Do you think that the explanation in the Understanding document helps with this?

**Comment by kerstinp** [comment]
@awkawk An explanation in the Understanding would help, but would just be informative. On the other hand I see the problem that an entry in the glossary would be an entry of the normative text of WCAG 2.0. In case a glossary entry is not possible would it be possible providing the explanation in the Understanding and in the [Errata](https://www.w3.org/WAI/WCAG20/errata/)?

**Comment by awkawk** [comment]
@kerstinp We are marking this as "implementation follow-up" - we can add some text to the understanding document to clarify this, but as it doesn't impact the normative text of WCAG 2.1 the issue is being closed (but the follow up label is used to ensure that we look at it again).
